frontend/src/org/components/EventForm/steps/PropagationStep.tsx

```diff
@@ -240,14 +241,19 @@ export const PropagationStep = ({
           </FormSection>
           <FormSection
             header="Web o akci"
-            help="Možnost přidat odkaz na webovou stránku vaší akce."
+            help='Možnost přidat odkaz na webovou stránku vaší akce. Odkaz vkládejte včetně "https://"'
             onWeb
           >
             <FormInputError>
               <input
                 type="url"
                 id="propagation.web_url"
-                {...register('propagation.web_url', {})}
+                {...register('propagation.web_url', {
+                  pattern: {
+                    value: /^(https?):\/\/[^\s/$.?#].[^\s]*$/,
```

Comment:
Překvapuje, že tuhle validaci nikde nemáme. Přidal bych to do souboru `helpers.ts` ať to neduplikujeme, pokud budeme někde řešit validaci URL znova.

Reply:
máš pravdu, už se to řeší jinde jinak, jen jsem to předtím nedopátrala a všimla jsem si toho na tvůj popud až teď. Změnila jsem to tedy ať je to konzistentní.

Reply:
Super, ještě by bylo úplně ideální tuhle funkcionalitu ze všech míst vytáhnout a udělat z toho pomocnou validační funkci. :)

Reply:
pokus číslo dvě - funkce přidána do helpers.ts